src/Files.App/ViewModels/Settings/DevToolsViewModel.cs

```diff
@@ -9,10 +9,16 @@ public sealed partial class DevToolsViewModel : ObservableObject
 	{
 		protected readonly IFileTagsSettingsService FileTagsSettingsService = Ioc.Default.GetRequiredService<IFileTagsSettingsService>();
 		protected readonly IDevToolsSettingsService DevToolsSettingsService = Ioc.Default.GetRequiredService<IDevToolsSettingsService>();
+		private readonly ICommonDialogService CommonDialogService = Ioc.Default.GetRequiredService<ICommonDialogService>();
 
 		public Dictionary<OpenInIDEOption, string> OpenInIDEOptions { get; private set; } = [];
 		public ICommand RemoveCredentialsCommand { get; }
 		public ICommand ConnectToGitHubCommand { get; }
+		public ICommand StartEditingIDECommand { get; }
+		public ICommand CancelIDEChangesCommand { get; }
+		public ICommand SaveIDEChangesCommand { get; }
+		public ICommand OpenFilePickerForIDECommand { get; }
+		public ICommand TestIDECommand { get; }
```

Comment:
Is this supposed to be removed before merging the PR?

Reply:
Currently not, why do you think we should remove them? 🤔

Reply:
My understanding was that `TestIDECommand` was to make it easier to debug without having to constantly reopen the settings dialog.

Reply:
Ideally yes, but I'm not sure how we should handle the testing: we need to launch the editor to see if it works, and it's not a good idea to kill the process (as it is now - I can change the code to get the ID of the launched  process). We cannot test it at each input event (obviously).
Should we test it only on save?